### DISCUSSION TITLE
Roadmap discussion

### DISCUSSION THREAD

**Discussion by EttingerK** [eclipse-sw360/sw360#2082] [General]
This is a discussion to harmonize on the different priorities of the different organizations.

**Comment by EttingerK** [comment]
Siemens AG high level priorities:

Stabilizing the system
- Update the archtecture (UI an endpoints)
- Restructuring backend
- Logger for restcalls
- being able handle big data (e.g. projects with 10k releases

Management of data
- integrating of a dashboard
- License and Obligation DB, to be used by SW360 and FOSSology

Improve of Clearing Performance 
- minimize outside automation
- integrated automation with FOSSology, e.g. compatibility agent

**Comment by KoukiHama** [comment]
My idea (not my company's, but some my peers agreed) for high-level priorities:

- Update the architecture (UI an endpoints) : same with Simense
  - Next to UI, test automation is also needed.

- For authentication, LDAP/SAML support is needed.

  - If LDAP and SAML are available, there is no need to use keyCloack.
     - but Keycloack is one candidate.

- Management of data
     - Consistency between Cyclone DX and SPDX
     - Consistency between package features and dependency network features

**Reply by KoukiHama** [reply]
I forgot to mention one.

Preparing document especially about new features is also needed for all sw360 users.

**Comment by EttingerK** [comment]
This is the today discussed Rodmap to rollout the new architecture:
![image](https://github.com/user-attachments/assets/b8fb56e3-9ec6-43ae-bbca-c46139b82279)

**Reply by KoukiHama** [reply]
Thank you! @EttingerK

**Reply by KoukiHama** [reply]
@tienlee @shi9qiu  @hoangnt2 
I think that you can use this image for sw360 in your company. 
and if you have comments about this, please let us know here or in the telco.

**Reply by shi9qiu** [reply]
Yes. It helps a lot.

**Reply by KoukiHama** [reply]
@EttingerK 

Are you able to access this Project page for roadmap?
https://github.com/eclipse-sw360/sw360/projects?query=is%3Aopen

![image](https://github.com/user-attachments/assets/9e8629ea-1b81-4143-b429-765af267f546)

If not, please update latest roadmap image here .
Then I will make new road map page base on the image.

**Reply by EttingerK** [reply]
Hi Kouki, sorry no I cannot access it, the green buttion is missing.
Perhaps in the next time I could be proposed as a contributor.
I send you the roadmap image here and thanks a lot for your support.

**Reply by EttingerK** [reply]
![image](https://github.com/user-attachments/assets/b03ebc05-926b-4c07-9d8e-5a867c8466ac)

**Reply by KoukiHama** [reply]
@EttingerK 

I made Roadmap with image.

https://github.com/orgs/eclipse-sw360/projects/8

![image](https://github.com/user-attachments/assets/0a454161-3577-40d9-8d7b-c78f227dfc7f)

**Reply by EttingerK** [reply]
Thanks a lot Kouki :-)
